02-oop-inheritance/Exercise 8 finished/js/EventEmitter.js

```diff
@@ -0,0 +1,20 @@
+export default class EventEmitter {
+
+    constructor() {
+        this.eventsMap = new Map();
+    }
+
+    on(event,instructions) {
+        if (!this.eventsMap.has(event)) {
```

Comment:
What happen if I have already a listener set and I want to replace it with another?